sklearn/linear_model/stochastic_gradient.py

```diff
@@ -732,6 +732,8 @@ class SGDClassifier(BaseSGDClassifier):
         depending on the number of samples already seen. Calling ``fit`` resets
         this counter, while ``partial_fit`` will result in increasing the
         existing counter.
+        
+        See :term:`the Glossary <warm_start>`.
```

Comment:
I feel like the preceding paragraph is more a note of details, and this should appear before it.